guides/v2.2/javascript-dev-guide/javascript/js-resources.md

```diff
@@ -136,6 +136,74 @@ To be available for the entire Magento instance, RequireJS library is included i
 
 * For the `frontend` area the similar configuration is located in [`app/code/Magento/Theme/view/frontend/layout/default.xml`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Theme/view/frontend/layout/default.xml).
 
+### Including third-party JavaScript Libraries
+
+For including a 3rd party library and use it in all the entire website, check the following steps where will be including the [Slick JS Library](https://github.com/kenwheeler/slick/) as an example. 
+
+1. Download the library and copy the `slick.min.js` to `<theme_path>/web/js` folder
+
+2. Copy the `slick.less` and `slick-theme.less` to `<theme_path>/web/css/source` folder. It is also required to include both files to `<theme_path>/web/css/source/_extend.less`.
+```less
+@import "slick.less";
+@import "slick-theme.less";
+```
+
+3. Create or update the theme's `requirejs-config.js` file
+> `<theme_path>/requirejs-config.js`
+
+```javascript
+var config = {
+    paths: {
+        slick: 'js/slick.min'
+    },
+    shim: {
+        slick: {
+            deps: ['jquery']
+        }
+    }
+};
+```
+
+After all these steps, clear the cache and deploy the files.
```

Comment:
Please, clarify this part slightly
```suggestion
After all these steps, clear the cache and perform the static content deployment
```

Reply:
Thank you for the suggestion, just updated it.